core/controllers/collection_editor.py

```diff
@@ -155,6 +140,25 @@ def _require_valid_version(self, version_from_payload, collection_version):
                 % (collection_version, version_from_payload))
 
     @require_editor
+    def get(self, collection_id):
+        """Populates the data on the individual collection page."""
+
+        try:
+            # Try to retrieve collection
+            collection_dict = (
+                summary_services.get_learner_collection_dict_by_id(
+                    collection_id, self.user_id,
+                    allow_invalid_explorations=True))
+        except Exception as e:
+            raise self.PageNotFoundException(e)
+
+        self.values.update({
+            'collection': collection_dict
```

Comment:
Why not do: ``self.values.update(collection_dict)``?

Since there's only one element in the dict, we could just send over the dict in-place. See my comment below for the put method, as it's similar.